ionic2/app/services/api.service.ts

```diff
@@ -284,4 +284,30 @@ export class ApiService {
     });
     return request.map(ApiService.handleResponse(Sighting));
   }
+
+  getTypes(): string[] {
+    const types: string[] = [
```

Comment:
If I move it out of the service, where should I put it? Another service called ConstantDataService or something?

I did put it there because it might get moved into the API, and because this apiservice is currently available anyway there (which might change ofc)